src/reactapp/public/index.html

```diff
@@ -49,7 +49,7 @@
                 <div
                     id="react-checkout"
                     class="checkout-container"
-                    data-checkout_config="{}" />
+                    data-checkout_config='{"language":"en_US"}' />
```

Comment:
conflicts with the PR I just reviewed I think

Reply:
never mind, I guess you added that branch to this one. I'll only look at the first commit

Reply:
yes, you are right.